packages/libs/react-ui/src/components/Form/TextField/TextField.tsx

```diff
@@ -1,29 +1,58 @@
 import type { IFormFieldWrapperProps, IInputProps } from '@components/Form';
-import { FormFieldWrapper, Input } from '@components/Form';
+import { Input } from '@components/Form';
+import { Stack } from '@components/Layout';
 import type { FC } from 'react';
 import React from 'react';
+import { FormFieldHeader } from '../FormFieldWrapper/FormFieldHeader/FormFieldHeader';
+import { FormFieldHelper } from '../FormFieldWrapper/FormFieldHelper/FormFieldHelper';
+import { statusVariant } from '../FormFieldWrapper/FormFieldWrapper.css';
 
 export interface ITextFieldProps
-  extends Omit<IFormFieldWrapperProps, 'children' | 'htmlFor'> {
-  inputProps: Omit<IInputProps, 'disabled' | 'children' | 'leadingTextWidth'>;
-}
-
+  extends Omit<IFormFieldWrapperProps, 'children' | 'htmlFor'>,
+    Omit<IInputProps, 'disabled' | 'children'> {}
+// FormField with action buttons? pass the children as its by default?
 export const TextField: FC<ITextFieldProps> = ({
   disabled = false,
-  inputProps,
   status,
+  leadingText,
+  startIcon,
+  leadingTextWidth,
+  type,
+  ref,
+  id,
+  outlined,
+  label,
+  info,
+  tag,
+  helperText,
   ...rest
 }) => {
-  const { id } = inputProps;
+  const statusVal = disabled === true ? 'disabled' : status;
 
   return (
-    <FormFieldWrapper
-      htmlFor={id}
-      disabled={disabled}
-      status={status}
-      {...rest}
-    >
-      <Input disabled={disabled} {...inputProps} />
-    </FormFieldWrapper>
+    <div className={statusVal ? statusVariant[statusVal] : undefined}>
+      {label !== undefined && (
+        <FormFieldHeader htmlFor={id} label={label} tag={tag} info={info} />
```

Comment:
This only passes `htmlFor`. You'll want to make sure to pass all react-aria props to the header (I think. Not sure if it should be only the title).